client/src/pages/ChatPage.js

```diff
@@ -0,0 +1,315 @@
+            type="submit"
+            onClick={exitUserHandler}
+            className="flex bg-indigo-300 ml-5 mb-2"
+          >
+            <ArrowRightOnRectangleIcon className="h-5 w-5" />
+          </IconButton>
+        </div>
+
+        <div className="flex justify-center">
+          <div className="mb-5 absolute bottom-0 w-3/5">
+            <Card className="flex mb-5">
+              <ul className="p-5 max-h-[30rem] overflow-y-auto scrollbar-width:none">
+                {chat.history
+                  ? chat.history.map((message, index) => (
+                      <li key={index}>
+                        <span className="font-bold">{message.username}: </span>
+                        {message.message}
+                      </li>
+                    ))
+                  : null}
+              </ul>
+            </Card>
+            <Card className="flex">
+              <form onSubmit={handleSubmit} className="flex p-5">
+                <Textarea
+                  label="Message"
+                  value={message}
+                  onChange={(event) => setMessage(event.target.value)}
+                />
+
+                <div className="flex items-center">
+                  <IconButton type="submit" className="flex bg-indigo-300 ml-5">
+                    <PaperAirplaneIcon className="h-3 w-3" />
+                  </IconButton>
+                </div>
+              </form>
+            </Card>
+          </div>
+        </div>
+      </div>
+
+      <Dialog
+        size="sm"
+        open={openUser}
+        handler={addUserHandler}
+        className="bg-transparent shadow-none"
+      >
+        <Card className="mx-auto w-full max-w-[24rem]">
+          <CardHeader
+            variant="gradient"
+            className="mb-4 grid h-28 place-items-center bg-indigo-300"
+          >
+            <Typography variant="h3" color="white">
+              Invite Friend
+            </Typography>
+          </CardHeader>
+          <CardBody className="flex flex-col gap-4">
+            <Input label="Username" size="lg" onChange={onInviteUser} />
+          </CardBody>
+          <CardFooter className="pt-0">
+            <Button
+              className="bg-indigo-300"
+              onClick={inviteDispatch}
+              fullWidth
+            >
+              Invite
+            </Button>
+          </CardFooter>
+        </Card>
+      </Dialog>
+
+      <Dialog
+        size="sm"
+        open={openName}
+        handler={changeNameHandler}
+        className="bg-transparent shadow-none"
+      >
+        <Card className="mx-auto w-full max-w-[24rem]">
+          <CardHeader
+            variant="gradient"
+            className="mb-4 grid h-28 place-items-center bg-indigo-300"
+          >
+            <Typography variant="h3" color="white">
+              Change Group Name
+            </Typography>
+          </CardHeader>
+          <CardBody className="flex flex-col gap-4">
+            <Input
```

Comment:
Maybe pre-populate the change chat name input with the existing name? Also, the dialog can be submitted even when the form is empty which effectively does nothing since it results in a 500 error:

![image](https://github.com/awctw/study-dash/assets/56493743/27bdbb17-f89f-47c8-b755-8b6e682af8c7)

Reply:
fixed! pushing soon..